cpp/src/arrow/symbols.map

```diff
@@ -33,6 +33,8 @@
     # Also export C-level helpers
     arrow_*;
     pyarrow_*;
+    # ARROW-14771: export Protobuf symbol table
+    descriptor_table_Flight_2eproto;
```

Comment:
Is this symbol created by protobuf?
Given `2e` is ascii of `.`, guess it might mean descriptor table of file `Flight.proto`, looks the naming should be stable.

Reply:
Yes, it's an internal symbol. I don't like dealing with implementation details like this, but hopefully this is reasonably stable, and it appears Protobuf relies on it.